backend/routes/deepface_routes.py

```
@@ -0,0 +1,1309 @@
+                stream_id, exc,
+            )
+
+    # Remove go2rtc relay stream
+    if settings.GO2RTC_ENABLED:
+        try:
+            async with httpx.AsyncClient(timeout=5) as rtc_client:
+                await rtc_client.delete(
+                    f"{settings.GO2RTC_API_URL}/api/streams?src={quote(stream_id, safe='')}",
+                )
+                logger.info("go2rtc stream removed: %s", stream_id)
+        except Exception as exc:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)